### PR TITLE
feat(meta): add title description and favicon

### PR DESCRIPTION
**Pull request by philibea**
## Type

- Feature

#### What is expected?

Add meta

### CODE REVIEW

**Review by matthprost — APPROVED** [review]

File: src/components/Head.tsx [hunk 1]
```diff
@@ -4,9 +4,24 @@ import NextHead from 'next/head'
...
   return (
     <NextHead>
-      <title>Create Next App</title>
-      <meta name="description" content="Generated by create next app" />
-      <link rel="icon" href="/favicon.ico" />
+      <meta charSet="UTF-8" />
+      <title> Scaleway UI</title>
+      <meta name="description" content="Open Source Component Library" />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Asap:wght@400;500;700&display=swap"
```
[the remark this answers is not in the document]
Comment (philibea) [reply src/components/Head.tsx]: Doesn't seem to work well yet

[the remark this answers is not in the document]
Comment (philibea) [reply src/components/Head.tsx]: Never mind it's work as expected

File: next.config.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const withTM = require('next-transpile-modules')([
 
 module.exports = withTM({
   poweredByHeader: false,
+  optimizeFonts: true,
```
Comment (chambo-e) [inline next.config.js:9]: Already default no ?

File: src/components/Head.tsx [hunk 3]
```diff
@@ -4,9 +4,12 @@ import NextHead from 'next/head'
 const Head = (): JSX.Element => {
   return (
     <NextHead>
-      <title>Create Next App</title>
-      <meta name="description" content="Generated by create next app" />
-      <link rel="icon" href="/favicon.ico" />
+      <meta charSet="UTF-8" />
```
Comment (chambo-e) [inline src/components/Head.tsx:7]: Should be in Document no ?

File: src/components/Head.tsx [hunk 4]
```diff
@@ -4,9 +4,12 @@ import NextHead from 'next/head'
...
+      <title> Scaleway UI</title>
```
Comment (chambo-e) [inline src/components/Head.tsx:8]: Why the extra space ?

**Review by chambo-e — APPROVED** [review]

**Pull request merged**